cmd/podman/top.go

```diff
@@ -24,38 +25,43 @@ Format Descriptors:
 }
 
 var (
-	topFlags = []cli.Flag{
-		LatestFlag,
-		cli.BoolFlag{
-			Name:   "list-descriptors",
-			Hidden: true,
-		},
-	}
+	topCommand     cliconfig.TopValues
 	topDescription = fmt.Sprintf(`Display the running processes of the container.  Specify format descriptors
 to alter the output.  You may run "podman top -l pid pcpu seccomp" to print
 the process ID, the CPU percentage and the seccomp mode of each process of
 the latest container.
 %s
 `, getDescriptorString())
 
-	topCommand = cli.Command{
-		Name:           "top",
-		Usage:          "Display the running processes of a container",
-		Description:    topDescription,
-		Flags:          sortFlags(topFlags),
-		Action:         topCmd,
-		ArgsUsage:      "CONTAINER-NAME [format descriptors]",
-		SkipArgReorder: true,
-		OnUsageError:   usageErrorHandler,
+	_topCommand = &cobra.Command{
+		Use:   "top",
+		Short: "Display the running processes of a container",
+		Long:  topDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			topCommand.InputArgs = args
+			topCommand.GlobalFlags = MainGlobalOpts
+			return topCmd(&topCommand)
+		},
+		Example: "CONTAINER-NAME [format descriptors]",
 	}
 )
 
-func topCmd(c *cli.Context) error {
+func init() {
+	topCommand.Command = _topCommand
+	flags := topCommand.Flags()
+	flags.BoolVar(&topCommand.ListDescriptors, "list-descriptors", false, "")
```

Comment:
No description?

Reply:
yeah, on hidden flags there are no descriptions in master,